### ISSUE TITLE
Hand or wrist mass 

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#10843]
**Preferred term label:**
Hand or wrist mass 

**Synonyms**

**Definition (free text, please give PubMed ID)**
Lump on the hand or wrist; bump on the hand or wrist; hand growth; wrist growth; wrist nodule; hand nodule; hand tumor; wrist tumor; wrist cyst; hand cyst

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Giant cell tumor of the tendon sheath (GCTTS)
Lipoma

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
It is not really possible in the HPO to say "X or Y mass", we would need separate terms. For now I am making a hand mass term, please reopen if we also need a wrist version.!

Adding new term: 
- ID: HP:6001437
- label: Hand mass
- parents: [Abnormal hand morphology (HP:0005922)](https://hpo.jax.org/app/browse/term/HP:0005922)
- pmid: 
- def: A swelling or enlargement localized to the hand. The word mass is usually used at an early stage of the diagnostic workup before the precise nature of the swelling has been determined.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/10843
- synonyms:   (Added all of the above hand synonyms)
